tap_salesforce/salesforce/__init__.py

```diff
@@ -381,13 +384,16 @@ def _get_selected_properties(self, catalog_entry):
                                             self.select_fields_by_default)]
 
 
-    def get_start_date(self, state, catalog_entry):
+    def get_start_date(self, state, catalog_entry, with_lookback=True):
```

Comment:
Change the default value of `with_lookback` to False and explicitly pass True as an argument wherever required. it will give more clarity regarding where lookback is used rather than not used.

Reply:
Updated.